pkg/csi/service/wcpguest/controller.go

```diff
@@ -531,6 +573,79 @@ func (c *controller) ControllerUnpublishVolume(ctx context.Context, req *csi.Con
 	return &csi.ControllerUnpublishVolumeResponse{}, nil
 }
 
+// controllerUnpublishForFileVolume is helper method to handle ControllerPublishVolume for File volumes
+func controllerUnpublishForFileVolume(ctx context.Context, req *csi.ControllerUnpublishVolumeRequest, c *controller) (
+	*csi.ControllerUnpublishVolumeResponse, error) {
+
+	log := logger.GetLogger(ctx)
+
+	// Check the feature state for file volume support
+	if commonco.ContainerOrchestratorUtility.IsFSSEnabled(ctx, common.FileVolume) {
```

Comment:
nit: This check can be moved to calling function, and then invoke `controllerUnpublishForFileVolume` if FSS is enabled. Just for better readability.

Reply:
Makes sense